pkg/kubectl/cmd/drain.go

```diff
@@ -140,6 +140,9 @@ var (
 		Drain node in preparation for maintenance.
 
 		The given node will be marked unschedulable to prevent new pods from arriving.
+		'drain' evicts the pods if the APIServer supports eviciton
+		(http://kubernetes.io/docs/admin/disruptions/). Otherwise, it will use normal DELETE
+		to delete the pods.
 		The 'drain' deletes all pods except mirror pods (which cannot be deleted through
```

Comment:
Above it says either evict or delete pods, but here it says deletes pods -- would that be confusing?